### ISSUE TITLE
Tried to register two views with the same name ARTSurfaceView | Android

### ISSUE THREAD

**Issue by SachArora** [tokkozhin/react-native-neomorph-shadows#35]
Throwing this error when I'm using this library in Android

Package.json

"dependencies": {
    "@react-native-community/async-storage": "^1.2.2",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-community/netinfo": "^2.0.10",
    "@react-native-firebase/analytics": "^7.1.5",
    "@react-native-firebase/app": "^7.3.0",
    "@react-native-firebase/crashlytics": "^7.1.6",
    "@react-native-firebase/in-app-messaging": "^7.1.5",
    "@react-native-firebase/messaging": "^7.1.7",
    "@react-native-firebase/remote-config": "^7.1.5",
    "asap": "^2.0.6",
    "axios": "^0.19.2",
    "cashfreereactnativepg": "^2.0.1",
    "geolib": "^3.2.1",
    "jwt-decode": "^2.2.0",
    "lottie-react-native": "^3.3.1",
    "moment": "^2.24.0",
    "native-base": "^2.12.1",
    "prop-types": "^15.7.2",
    "react": "^16.8.6",
    "react-devtools": "^3.6.3",
    "react-native": "^0.59.8",
    "react-native-aes-crypto": "^1.3.6",
    "react-native-android-fullscreen-webview-video": "^1.0.4",
    "react-native-android-installed-apps": "^1.0.1",
    "react-native-android-location-enabler": "^1.2.0",
    "react-native-animate-loading-button": "^1.0.3",
    "react-native-aws3-cognito": "0.0.2",
    "react-native-confetti": "^0.1.0",
    "react-native-config": "^0.12.0",
    "react-native-confirmation-code-field": "^4.1.0",
    "react-native-datepicker": "^1.7.2",
    "react-native-device-info": "^5.5.3",
    "react-native-document-picker": "^2.3.0",
    "react-native-draggable-flatlist": "^1.1.7",
    "react-native-event-listeners": "^1.0.3",
    "react-native-extended-stylesheet": "^0.12.0",
    "react-native-fast-image": "^7.0.2",
    "react-native-fbsdk": "^0.8.0",
    "react-native-flip-card": "^3.5.5",
    "react-native-floating-labels": "^1.1.9",
    "react-native-gesture-handler": "^1.6.1",
    "react-native-get-location": "^1.2.0",
    "react-native-google-signin": "^1.2.3",
    "react-native-htmlview": "^0.15.0",
    "react-native-image-picker": "^0.28.1",
    "react-native-image-view": "^2.1.4",
    "react-native-intercom": "^13.2.0",
    "react-native-is-device-rooted": "^1.0.3",
    "react-native-linear-gradient": "^2.5.4",
    "react-native-linkedin": "^1.3.2",
    "react-native-loading-spinner-overlay": "^1.0.1",
    "react-native-material-dropdown": "^0.11.1",
    "react-native-media-meta": "0.0.10",
    "react-native-multiple-select": "^0.5.5",
    "react-native-neomorph-shadows": "^1.1.2",
    "react-native-orientation": "^3.1.3",
    "react-native-paper": "^2.15.1",
    "react-native-phone-call": "^1.0.9",
    "react-native-pinch-zoom-view": "^0.2.0",
    "react-native-popup-dialog": "^0.18.2",
    "react-native-progress": "^3.6.0",
    "react-native-quick-actions": "^0.3.13",
    "react-native-rate": "^1.2.1",
    "react-native-razorpay": "^2.1.25",
    "react-native-reanimated": "^1.8.0",
    "react-native-safe-area-context": "^0.7.3",
    "react-native-screens": "^2.7.0",
    "react-native-share": "^1.1.3",
    "react-native-shimmer-placeholder": "^1.0.35",
    "react-native-simple-shadow-view": "^1.6.3",
    "react-native-simple-toast": "0.0.8",
    "react-native-svg": "^9.4.0",
    "react-native-svg-charts": "^5.2.0",
    "react-native-swiper": "^1.5.14",
    "react-native-tab-view": "^2.7.1",
    "react-native-vector-icons": "^6.4.2",
    "react-native-video": "^4.4.1",
    "react-native-walkthrough-tooltip": "^1.1.7",
    "react-native-webview": "^6.8.1",
    "react-navigation": "^4.3.8",
    "react-navigation-drawer": "^1.4.0",
    "react-navigation-stack": "^1.10.3",
    "react-navigation-tabs": "^1.2.0",
    "react-navigation-transitions": "^1.0.10",
    "react-redux": "^7.1.0",
    "redux": "^4.0.4",
    "redux-logger": "^3.0.6",
    "redux-thunk": "^2.3.0",
    "smartlook-react-native-wrapper": "^0.30.0",
    "socket.io-client": "^2.1.1"
  }

RN 0.59.8

**Comment by tokkozhin** [comment]
Hi! First of all, you need to install the '@react-native-community/art'. I don't see it in your package.

**Comment by SachArora** [comment]
@tokkozhin yes I know. even after that same thing happens because my current RN version is 0.59. In 0.59 ART is accessible through the react-native package. If I tried to import from @react-native-community/art this happens. I guess on the native side both of them are using something same. How to get rid of it without migrated to the latest version.

**Comment by Gopalakrishnan-V** [comment]
Any update on this? @tokkozhin
